db/migrate/20171212130557_add_indexes_to_subscriber_list.rb

```diff
@@ -0,0 +1,9 @@
+class AddIndexesToSubscriberList < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
```

Comment:
Why are we disabling transactionality? If the migration fails that would put us in a weird state.

Reply:
It's so we can add the indexes concurrently without locking the table. If the migration fails, we can just run it again.